portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro

```diff
@@ -187,7 +191,7 @@ definition {
 			AssertElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
 		}
 		else {
-			User.loginPG(
+			User.firstLoginUI(
```

Comment:
hey @gaborlovasdotcom ,
I think we need change:
specificURL = "${specificURL}",
to 
specificURL = "${spURL}",

Reply:
I'm reviewing all URL namings, wip.